react/components/ManageChannel/index.js

```diff
@@ -141,9 +141,12 @@ class ManageChannel extends Component {
 
     if (loading) return <LoadingIndicator />;
 
-    const { data: { channel, me: { counts } } } = this.props;
+    const { data: { channel, me, me: { counts } } } = this.props;
     const owner = `${channel.owner.__typename.toUpperCase()}:${channel.owner.id}`;
 
+    // Only the original author can change the owner;
+    const showAuthor = counts.groups > 0 && me.id === channel.user.id;
```

Comment:
This is fine for the quick fix but should likely be a `can`

Reply:
Indeed